### DISCUSSION TITLE
Possible renaming?

### DISCUSSION THREAD

**Discussion by sonakravtsenko3** [thygate/stable-diffusion-webui-depthmap-script#306] [Ideas]
@thygate I think we would need to get a new name for the (script?, program?, tool?, webui? generator? - need to decide on how to call this now) this repository hosts. Standalone mode is a thing now (needs some extra features, but quite usable), advertise-able API will quickly follow, and this repo knows a lot of extra tricks, not only how to generate "pure"/"vanilla" depthmaps.

This potentially could be a new name for the program only, or also renaming of the whole repository. The latter of course would be much harder - no idea how to make the updating work seamlessly with Automatic WebUI, and also we would probably lose Google rating and what not. But the former would look slightly weird.

**Comment by graemeniedermayer** [comment]
I think it's a bad idea to do this too quickly/rashly. There's a lot of youtube videos advertising this exact repo + the google rating search-ability metioned. A repo name change might break url links.

Also in the past few months auto1111 + this repo have been doing quite well. Auto1111 webui is the now like the 50th most starred repo on all github. It's about to past typescript.

I do think the standalone we should start referring to as the "Speedier lower vram standalone".

**Comment by thygate** [comment]
I thought of renaming a few times, but did not do so because it would break all the links here. 
The stable diffusion part in the title does conflict somewhat with the new option to run in standalone mode, and depthmaps is not the only thing it does, perhaps we should just make this more obvious in the readme title and intro. I would prefer not renaming the repo.

**Comment by sonakravtsenko3** [comment]
What about a new and catchy name that we could use (instead of simply DepthMap or stable-diffusion-webui-depthmap-script? Something that we will put in bold letters at the top of the README file and as a title for standalone mode. Something on the likes of ExtraDepth, ExtraDeep, or VolumetricGen (just some examples, I hope there could be something better and more fitting).

**Reply by sonakravtsenko3** [reply]
@thygate What do you think?

**Comment by graemeniedermayer** [comment]
I do think the Depthmap Examples are starting to look a little dated (low res). We should set up a friendly competition for a replacement. This could highlight all the recent additions.

**Reply by sonakravtsenko3** [reply]
Definitely. This is one part of https://github.com/thygate/stable-diffusion-webui-depthmap-script/pull/297
Stereoimages are actually broken 😅
Interesting idea about friendly competition :)

By the way, I think all images should be put to wiki, and README should have links to the wiki. This is because every image ever present in the repo is downloaded on pull.
